### DISCUSSION TITLE
New themes request: Material and Cupertino

### DISCUSSION THREAD

**Discussion by Muhammed-Rahif** [saadeghi/daisyui#3209] [Ideas / feature requests]
https://www.material-tailwind.com is out there, but what about daisyui implements material and cupertino themes/components? Is that possible? 
Getting full features of material/cupertino will not be possible with tailwind css only though, we can make the existing components look like them by adding new themes, right?

**Comment by saadeghi** [comment]
You can make your own theme here and control all the colors.
https://daisyui.com/theme-generator/

Change all colors to black and white and it will be the the one you asked:  
https://play.tailwindcss.com/9Guz3tfVmA?file=config
